content/200-concepts/100-components/04-introspection.mdx

```diff
@@ -175,6 +146,22 @@ model User {
 }
 ```
 
+Trying to generate your Prisma Client with `prisma generate` you would get the following error:
+
+```
+$ npx prisma generate
+
+Error: Schema parsing
+error: The model "User" cannot be defined because a model with that name already exists.
+  -->  schema.prisma:17
+   |
+16 | }
+17 | model User {
+   |
+
+Validation Error Count: 1
+```
```

Comment:
````suggestion
<CodeWithResult expanded={true}>

<cmd>

```
npx prisma generate
```

</cmd>

<cmdResult>

```
$ npx prisma generate
Error: Schema parsing
error: The model "User" cannot be defined because a model with that name already exists.
  -->  schema.prisma:17
   |
16 | }
17 | model User {
   |

Validation Error Count: 1
```

</cmdResult>

</CodeWithResult>
````

Like this @molebox?

Reply:
Yep, that looks correct to me!